packages/core/src/transport/httpRequest.spec.ts

```diff
@@ -21,7 +27,7 @@ describe('httpRequest', () => {
     request.send('{"foo":"bar1"}\n{"foo":"bar2"}', 10)
 
     expect(server.requests.length).toEqual(1)
-    expect(server.requests[0].url).toContain(ENDPOINT_URL)
+    expect(server.requests[0].url).toContain(endpointBuilder.build())
```

Comment:
FMU, previous assertions should work and would avoid to write HttpRequest internal logic in the assertion.

Reply:
Interesting, I'll change it